src/main/java/org/dataconservancy/cos/osf/client/model/Wiki.java

```diff
@@ -141,23 +110,23 @@ public void setName(String name) {
         this.name = name;
     }
 
-    public String getMaterialized_path() {
+    public String getMaterializedPath() {
         return materialized_path;
     }
 
-    public void setMaterialized_path(String materialized_path) {
+    public void setMaterializedPath(String materialized_path) {
```

Comment:
These methods should be named after the name of the field: so the original name, setMaterialized_path is correct. That is why these values are not being set: the method name is incorrect.